common/src/main/java/com/redhat/cloud/notifications/models/event/TestEventHelper.java

```diff
@@ -0,0 +1,116 @@
+ * information, take a look at <a href="https://issues.redhat.com/browse/RHCLOUD-22470">RHCLOUD-22470</a>.
+ */
+public class TestEventHelper {
+    /**
+     * Regular action test data.
+     */
+    public static final String TEST_ACTION_CONTEXT_ENDPOINT_ID = "test-action-context-endpoint-id";
+    public static final String TEST_ACTION_METADATA_KEY = "test-metadata-key";
+    public static final String TEST_ACTION_METADATA_VALUE = "test-metadata-value";
+    public static final String TEST_ACTION_PAYLOAD_KEY = "message";
+    public static final String TEST_ACTION_PAYLOAD_VALUE = "Congratulations! The integration you created on https://console.redhat.com was successfully tested!";
+    public static final String TEST_ACTION_RECIPIENT = "test-recipient-1";
+    public static final String TEST_ACTION_VERSION = "0.0.0";
+    /**
+     * The test action or event will be sent with a defined "console" bundle, the "notifications" application and
+     * a new event type that has been inserted in the V1.71.0 migration.
+     */
+    public static final String TEST_ACTION_BUNDLE = "console";
+    public static final String TEST_ACTION_APPLICATION = "integrations";
+    public static final String TEST_ACTION_EVENT_TYPE = "integration-test";
+
+    /**
+     * Creates a test action ready to be sent to the engine. It sets the endpoint's UUID and a flag in the context, so
+     * that it can be easily identified.
+     * @param endpointUuid the endpoint UUID that will be set in the context.
+     * @param orgId the org ID for the action.
+     * @return the created action.
+     */
+    public static Action createTestAction(final UUID endpointUuid, final String orgId) {
+        Action testAction = new Action();
+
+        final var context = new com.redhat.cloud.notifications.ingress.Context();
+        context.setAdditionalProperty(TEST_ACTION_CONTEXT_ENDPOINT_ID, endpointUuid);
+        testAction.setContext(context);
+
+        /*
+         * Create a test event which will be sent along with the action.
+         */
+        Event testEvent = new Event();
+
+        Metadata metadata = new Metadata();
+        metadata.setAdditionalProperty(TEST_ACTION_METADATA_KEY, TEST_ACTION_METADATA_VALUE);
+
+        Payload payload = new Payload();
+        payload.setAdditionalProperty(TEST_ACTION_PAYLOAD_KEY, TEST_ACTION_PAYLOAD_VALUE);
+
+        testEvent.setMetadata(metadata);
+        testEvent.setPayload(payload);
+
+        Recipient recipient = new Recipient();
+        recipient.setUsers(List.of(TEST_ACTION_RECIPIENT));
+
+        /*
+         * Set the rest of the action's members.
+         */
+        testAction.setApplication(TEST_ACTION_APPLICATION);
+        testAction.setBundle(TEST_ACTION_BUNDLE);
+        testAction.setEvents(List.of(testEvent));
+        testAction.setEventType(TEST_ACTION_EVENT_TYPE);
+        testAction.setId(UUID.randomUUID());
+        testAction.setRecipients(List.of(recipient));
```

Comment:
We don't need recipients in this action as it is not meant to test email integations.
```suggestion
```

Reply:
Removed!